### PR TITLE
106 user presense resolverspec

### PR DESCRIPTION
**Pull request by Chicanoss**
Done for NanoIdtest

### CODE REVIEW

File: backend/src/user-presence/user-presence.resolver.spec.ts [hunk 1]
```diff
@@ -1,23 +1,224 @@
...
+      userPresenceService.findAllByUserId.mockReturnValue(resExpected)
+
+      const result = await userPresenceResolver.findAllUserPresenceByUserId(
+        '01'
+      )
+
+      expect(result).toStrictEqual(resExpected)
+      expect(userPresenceService.findAllByUserId).toHaveBeenCalledWith('01')
+    })
+    //////////////////
```
Comment (kbarbry) [inline backend/src/user-presence/user-presence.resolver.spec.ts:104]: ciao

File: backend/src/user-presence/user-presence.resolver.spec.ts [hunk 2]
```diff
@@ -1,23 +1,224 @@
...
+        }
+        const thrownError = await validationPipe
+          .transform(data, metadata)
+          .catch((error) => error)
+        expect(thrownError.getResponse()).toStrictEqual(res)
+      })
+    })
+  })
+})
 //check with invalid user
```
Comment (kbarbry) [inline backend/src/user-presence/user-presence.resolver.spec.ts:222]: ciao

File: backend/src/user-presence/user-presence.resolver.spec.ts [hunk 3]
```diff
@@ -1,23 +1,224 @@
...
+    })
+    //////////////////
+    it('isUserPresenceConnected', async () => {
+      userPresenceService.isConnected.mockReturnValue(true)
+      const res = await userPresenceResolver.isUserPresenceConnected('01')
+      expect(res).toStrictEqual(true)
+      expect(userPresenceService.isConnected).toHaveBeenCalledWith('01')
+    })
+  })
+
```
Comment (kbarbry) [inline backend/src/user-presence/user-presence.resolver.spec.ts:112]: test ValidationPipe

**Review by kbarbry — CHANGES_REQUESTED** [review]

**Review by Chicanoss — COMMENTED** [review]
Nice work, just check the validation pipe

**Review by Chicanoss — COMMENTED** [review]
Nice work

**Review by Isioblast — APPROVED** [review]

**Review by kbarbry — APPROVED** [review]

**Pull request merged**